src/Generator/ClassGenerator.php

```diff
@@ -776,11 +772,7 @@ public function removeUseAlias($use)
         return $this;
     }
 
-    /**
-     * Returns the "use" classes
-     *
-     * @return array
-     */
+    /** {@inheritDoc} */
```

Comment:
Same here:

```suggestion
    /** @inheritDoc */
```

Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.